scopes/design/ui/Avatar/avatar.docs.mdx

```diff
@@ -0,0 +1,46 @@
+---
+labels: ['react', 'typescript', 'avatar', 'icon']
+description: 'An avatar for all occasions'
+---
+
+import { ThemeCompositions } from '@teambit/documenter.theme.theme-compositions';
+import { DefaultAvatar, OrgAvatar, UserAvatar } from './index';
+
+
+### Overview
+
+An avatar component, consisting of 3 different types of Avatar:  
+
+1. Default Avatar - no icon, just an '?' and size  
+2. Organization Avatar - default org icon (see compositions)
+3. User Icon - icon with initials overlaid
+
+```js live
+() => {
+  const accounts = {
+  defAccount: { name: 'defaultAccount', type: 'default', profileImage: 'https://static.bit.dev/harmony/support.svg' },
+  orgAccount: { name: 'defaultAccount', type: 'organization', profileImage: 'https://static.bit.dev/bit-logo.svg' },
+  userAccount: { name: 'defaultAccount', type: 'user', profileImage: 'https://static.bit.dev/harmony/github.svg' },
+};
+
+ return (
+  <>
+    <ThemeCompositions style={{ minHeight: 90 }}>
+      Basic default avatar:
+      <DefaultAvatar size={32} account={accounts.defAccount} />
+    </ThemeCompositions>
+    <ThemeCompositions style={{ minHeight: 90}}>
+      Organisation avatar with icon
+      <OrgAvatar size={32} account={accounts.orgAccount} />
+    </ThemeCompositions>
+    <ThemeCompositions style={{ minHeight: 90}}>
+      User avatar
+      <UserAvatar size={32} account={accounts.userAccount} />
+
+    </ThemeCompositions>
+  </>
+ )
+}
+
+```
+
```

Comment:
You don't need to use ThemeCompositions to envelope each example, it's preferable to use it one time as the component that contained all the children.
And then override add a flexDirection to make the example in column.
I'll do these changes.